schedule/postpone.py

```diff
@@ -38,8 +38,8 @@ def postpone(did):
             ivl,
             json_extract(data, '$.s'),
             CASE WHEN odid==0
-            THEN {mw.col.sched.today} - (due - ivl)
-            ELSE {mw.col.sched.today} - (odue - ivl)
+            THEN {mw.col.sched.today} - (due - ivl) + ivl * 0.075
```

Comment:
What's the purpose of plus ivl * 0.075?

Reply:
I have the same question as well.

Reply:
It is used to get an estimate of the elapsed days at the new due date.

The code for the new interval is this:

```py
        delay = elapsed_days - ivl
        new_ivl = min(
            max(1, math.ceil(ivl * (1.05 + 0.05 * random.random())) + delay), max_ivl
        )
```

For the reason explained in https://github.com/open-spaced-repetition/fsrs4anki-helper/pull/407#issuecomment-2104584109, I assume this to be:

```py
        delay = elapsed_days - ivl
        new_ivl = min(
            max(1, math.ceil(ivl * (1.05 + 0.025)) + delay), max_ivl
        )
```

If you put the value of `delay`, you get
```py
        new_ivl = min(
            max(1, math.ceil(ivl * 0.075) + elapsed_days), max_ivl
        )
```

For the purpose of sorting, we can skip `math.ceil` and assume that the value is between `1` and `max_ivl`. So, we get `new_ivl = ivl * 0.075 + elapsed_days`

At due date, `R = power_forgetting_curve(new_ivl, stability)`

Reply:
Oh, interesting. That's a clever way to take fuzz into account.

Reply:
> assume that the value is between 1 and max_ivl

If you don't want to make this assumption, we can change it to something like

```py
THEN min({mw.col.sched.today} - (due - ivl) + ivl * 0.075, max_ivl)
```

Reply:
Yeah, that's better. ivl * 0.075 is more than 1 only if ivl is >13.3(3)

Reply:
Ok

Reply:
The value not being > 1 is not actually a problem because it is elapsed_days + ivl * 0.075 and elapsed_days would always be at least equal to 1 (you can't postpone a learning card).

The main concern is that this value can sometimes become greater than the max_ivl but the add-on would not assign that value of interval. So, the retention estimate would become inaccurate.